src/themes/hugo-theme-learn/layouts/partials/custom-header.html

```diff
@@ -2,4 +2,5 @@
 <style type="text/css">
     /* Custom css */
 </style>
--->
\ No newline at end of file
+-->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
```

Comment:
Comment above can be removed

Reply:
Fixed in https://github.com/PrestaShop/docs/pull/1047/commits/082c5757f14a946321a7a360cc3c8d73dc01b7cb